content/en/docs/reference/storage/_index.md

```diff
@@ -0,0 +1,359 @@
+---
+title: Storage Volumes
+weight: 145
+no_list: true
+---
+
+<!-- overview -->
+This reference contains references to different volume types.
+
```

Comment:
How about adding:
```suggestion
The page  lists details for volume types that are deprecated or removed. For details of volume
types that Kubernetes supports, see elsewhere in the Kubernetes documentation.

```

Alternative: also list those volume types here, even the supported ones.

Reply:
Yup. That can be done in a followup PR.

Reply:
I am worried we would get a slow but regular stream of issues about the list of volumes being incomplete.

With neither option from https://github.com/kubernetes/website/pull/43278/files#r1348880767 chosen, I recommend _not_ merging this PR. It's that important (in my opinion).

Reply:
This PR is about the deprecated and removed volumes. Isn't this clear enough to you?
The intent is to keep the list more accurate, rather than including everything here.

Reply:
>  This PR is about the deprecated and removed volumes. Isn't this clear enough to you?

_I_ know the intent here; readers won't.